README.md

```diff
@@ -1,72 +1,92 @@
-👉 *The official Documents for [The Crab Network](https://docs.crab.network/)*<br>
-👉 *The official communication zoom, if you have any question? [Darwinia Room](https://matrix.to/#/#darwinia:matrix.org)*<br>
+![Logo](https://github.com/darwinia-network/rfcs/raw/master/logo/darwinia.png)
+
+# Darwinia
+Implementation of a https://darwinia.network node in Rust based on the **Substrate** framework.
+
+This repository contains runtimes for the **[Darwinia](https://darwinia.network)** and **[Crab](https://crab.network)** networks.
+
+## Resources
+
+### Documents
+- [Darwinia Network Docs](https://docs.darwinia.network)
+- [Crab Network Docs](https://docs.crab.network)
+
+### Social Media
+- Telegram
+	- [Announcement Channel](https://t.me/DarwiniaNetwork_Announcement)
+	- [Official Group](https://t.me/DarwiniaNetwork)
+	- [Official Group (CN)](https://t.me/DarwiniaNetworkCN)
+	- [Official Technical Group](https://t.me/DarwiniaDev)
+	- [Unofficial Group](https://t.me/darwinianetworkpricespeculation)
+- Matrix
+	- [Official Technical Room](https://matrix.to/#/#darwinia:matrix.org)
+- Discord
+	- [Official Server](https://discord.gg/uqa3snSGTj)
+- Slack
+	> Technical members are more active on this place. If you expect a quick response, here you go.
+	- PM [Support](mailto:support@darwinia.network) or [Xavier](mailto:xavier.lau@itering.com)
+- Reddit
+	- [Community](https://www.reddit.com/r/DarwiniaCommunity)
+- Twitter
+	- [Official Account](https://twitter.com/DarwiniaNetwork)
+- Medium
+	- [Official Account](https://darwinianetwork.medium.com)
+- Wechat
+	- Official Group: Add "**DarwiniaBot**" to get the link
+	- Official Account: "**DARWINIA**"
+- Weibo
+	- [Official Account](https://weibo.com/DarwiniaNetwork)
+
+## Installation
+- Downloading pre-built binary from **[releases](https://github.com/darwinia-network/darwinia/releases)** page.
+- Using the docker image on **[releases](https://github.com/darwinia-network/darwinia/releases)** page.
+- Building from source follow this **[guide](#build-from-source)**.
 
 ## Building from Source
+> Make sure that you have the dependencies. Follow [substrate-getting-started](https://substrate.dev/docs/en/knowledgebase/getting-started).
 
-> NOTE: Substrate development for windows is not properly supported! It is highly recommend to use Windows Subsystem Linux (WSL) and follow the instructions for Ubuntu/Debian. Check out [Getting Started on Windows](https://substrate.dev/docs/en/knowledgebase/getting-started/windows-users) for more detail.
-
-For Unix-based user, just grab the source code and build it. Make sure you have Rust and the support software installed.
-
-Install Rust first:
-
+### Installing via Cargo
 ```sh
-curl https://sh.rustup.rs -sSf | sh
+cargo install --git https://github.com/darwinia-network/darwinia --tag <version> --locked
 ```
 
+### Building via Source
 ```sh
-rustup default nightly
-rustup target add wasm32-unknown-unknown
+gh repo clone darwinia-network/darwinia
```

Comment:
Suggest change to `git clone` . Maybe some readers didn't know what *gh repo clone* meant.